base/logging.jl

```diff
@@ -674,7 +674,7 @@ function handle_message(logger::SimpleLogger, level::LogLevel, message, _module,
     end
     iob = IOContext(buf, stream)
     levelstr = level == Warn ? "Warning" : string(level)
-    msglines = eachsplit(chomp(string(message)::String), '\n')
+    msglines = eachsplit(chomp(convert(String, message)::String), '\n')
```

Comment:
I'm the original source of the bug here. This and a similar change below in ConsoleLogger.jl should be backported to 1.8 (I'll submit a separate PR against that branch).